src/app/modules/workspace/workspace.controller.ts

```diff
@@ -301,3 +252,40 @@ export const moveFormsToWorkspace: ControllerHandler<
       return res.status(statusCode).json({ message: errorMessage })
     })
 }
+
+/**
+ * Handler for POST /workspace/remove
+ * @security session
+ *
+ * @returns 200 if form is successfully removed
+ * @returns 401 when user is not logged in
+ * @returns 500 when database errors occur
+ */
+export const removeFormsFromWorkspaces: ControllerHandler<
+  unknown,
+  void | ErrorDto,
+  { formIds: string[] }
+> = async (req, res) => {
+  const userId = (req.session as AuthedSessionData).user._id
+  const { formIds } = req.body
+
+  return WorkspaceService.removeFormsFromAllWorkspaces({
+    formIds,
+    userId,
+  })
+    .map(() => res.sendStatus(StatusCodes.OK))
+    .mapErr((error) => {
+      logger.error({
+        message: 'Error deleting forms from all workspaces',
+        meta: {
+          action: 'deleteFormFromWorkspaces',
```

Comment:
Should `meta.action` be `removeFormsFromWorkspaces` instead?

Reply:
Ahh thanks for catching all the logging inconsistencies 🙏

[b734172](https://github.com/opengovsg/FormSG/pull/6754/commits/b734172b81001f24780c9a374ae0ee1db95bdf97)